services/rds/src/main/java/org/finra/gatekeeper/services/aws/RdsLookupService.java

```diff
@@ -382,7 +410,71 @@ private List<GatekeeperRDSInstance> loadToGatekeeperRDSInstanceCluster(AWSEnviro
+        ArrayList<GatekeeperRDSInstance> gatekeeperRDSInstances = new ArrayList<>();
+        instances.forEach(item -> {
+            String application = getApplicationTagforRedshift(item.getTags());
+            boolean enabled = false;
+            String status = item.getClusterStatus();
+            Integer port = item.getEndpoint().getPort();
+            List<String> availableRoles = null;
+            String dbName = item.getDBName();
+            String address = item.getEndpoint().getAddress() + ":" + port;
+
+            if(item.getClusterStatus().equalsIgnoreCase(STATUS_AVAILABLE)
+                    || item.getClusterStatus().equalsIgnoreCase(STATUS_BACKING_UP)) {
+                enabled = item.getVpcSecurityGroups().stream()
+                        .anyMatch(sg -> securityGroupIds.contains(sg.getVpcSecurityGroupId()));
+
+                if(!enabled){
+                    status = STATUS_MISSING_SGS;
+                }
+
+                if(enabled) {
+                    try {
+                        String dbStatus = databaseConnectionService.checkDb(item, environment);
+                        status = !dbStatus.isEmpty() ? dbStatus : status;
+                        enabled = dbStatus.isEmpty(); // if there's no message back from the DB enabled is still true
+                    }catch(GKUnsupportedDBException e){
+                        logger.error(STATUS_UNSUPPORTED_DB_ENGINE, e);
+                        status = STATUS_UNSUPPORTED_DB_ENGINE;
+                        enabled = false;
+                    }
+
+                    if(enabled){
+                        // get available roles for DB
+                        try {
+                            availableRoles = databaseConnectionService.getAvailableRolesForDb(item, environment);
+                            Collections.sort(availableRoles);
+                            logger.info("Found the following roles on " + item.getClusterIdentifier() + " (" + availableRoles +").");
+                        } catch (Exception e) {
+                            logger.error(STATUS_COULD_NOT_FETCH_ROLES, e);
+                            if (e.getMessage().contains("password")) {
+                                status = STATUS_UNABLE_TO_LOGIN;
+                            } else {
+                                status = STATUS_COULD_NOT_FETCH_ROLES;
+                            }
+                            enabled = false;
+                        }
+                    }
+                }
+            }
+            String clusterArn = null;
+            try {
+                clusterArn = getRedshiftArn(environment, item.getClusterIdentifier());
+            } catch (GatekeeperException gatekeeperException) {
+                logger.error("Failed to construct clusterArn: " + gatekeeperException.getMessage());
```

Comment:
Can we call out the cluster identifier in the try block so we know where it got messed up, this will help for any future investigation when there is a problem.